pdns/dnsdistdist/doh.cc

```diff
@@ -2,6 +2,7 @@
 #include "doh.hh"
 
 #ifdef HAVE_DNS_OVER_HTTPS
+#ifdef HAVE_LIBH2OEVLOOP
```

Comment:
This entire file is now unused when libh2o is being used. Perhaps gating it in the build system instead of using the preprocessor would be a better option?

Reply:
Makes sense to me, I'll have a look!

Reply:
Done!